app/livechat/client/views/app/livechatReadOnly.js

```diff
@@ -0,0 +1,61 @@
+import { Meteor } from 'meteor/meteor';
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+import { ChatRoom } from '../../../../models';
+import { LivechatInquiry } from '../../../lib/LivechatInquiry';
+import { handleError } from '../../../../utils';
+import { call } from '../../../../ui-utils/client';
+import { settings } from '../../../../settings';
+import './livechatReadOnly.html';
+
+Template.livechatReadOnly.helpers({
+	inquiryOpen() {
+		const inquiry = Template.instance().inquiry.get();
+		return inquiry || FlowRouter.go('/home');
+	},
+
+	roomOpen() {
+		const room = Template.instance().room.get();
+		return room && room.open === true;
+	},
+
+	guestPool() {
+		return settings.get('Livechat_Routing_Method') === 'Guest_Pool';
+	},
+});
+
+Template.livechatReadOnly.events({
+	async 'click .js-take-it'(event, instance) {
+		event.preventDefault();
+		event.stopPropagation();
+
+		const inquiry = instance.inquiry.get();
+		const { _id } = inquiry;
+		try {
```

Comment:
if I'm not mistaken `call` already has a `handleError`you dont need to handle by yourself